src/actions/editor.ts

```diff
@@ -291,3 +307,29 @@ export function setEditorReference(editorRef: any) {
 }
 
 export type SetEditorReference = ReturnType<typeof setEditorReference>;
+
+export function setLogLevel(logLevel: string) {
+  return {
+    logLevel,
+    type: SET_LOG_LEVEL,
+  };
+}
+
+export type SetLogLevel = ReturnType<typeof setLogLevel>;
+
+export function setHover(hover: boolean) {
+  return {
+    hoverEnable: hover,
```

Comment:
Should we support true, false, and auto? I think we don't need hover for VegaLIte by default.